### PR TITLE
Small fixups

### PR DESCRIPTION
**Pull request by blag**
* [f428e8f](https://github.com/blag/django-tos/commit/f428e8fa49277c257b8b41912371f29a89370f9b) - The `next` template variable is the data *value*, and use `redirect_field_name` for the field *name*
* [4b6a510](https://github.com/blag/django-tos/commit/4b6a5108a7c46bc7f6c7764ffaa9cbe48bb57d50) - Using HTML buttons instead of inputs enables i18n without changing the field values
* [2383ad8](https://github.com/blag/django-tos/commit/2383ad837cd13b5ba7abe26c99e8a2a2c73af2f3) - Turn on i18n in templates
* [b0a7c04](https://github.com/blag/django-tos/commit/b0a7c0459b3d18807fcc881670319e8e6c7fa2f5) - The `/confirm` view allows users to accept user agreements multiple times, and every time a new UserAgreement is created pointing to the same `TermsOfService` and user (read: a duplicate row). Use `UserAgreement.objects.get_or_create()` instead of just `UserAgreement.objects.create()` to avoid creating duplicate `UserAgreements`.
* [3488d54](https://github.com/blag/django-tos/commit/3488d54a9eaf03bb57257a5e66156942a7287e08) - Add test for avoiding duplicate `UserAgreements`

### CODE REVIEW

**Comment by nicholasserra** [comment]
Looks like this needs a conflict resolved now that those other fixes are in. Hoping to take a look at this tomorrow. Thanks :D

File: tos/tests/test_middleware.py [hunk 1]
```diff
@@ -82,3 +84,70 @@ def test_accept_after_middleware_redirects_properly(self):
...
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url.replace('http://testserver', ''), str(reverse('index')))
+
+
+@override_settings(
+    MIDDLEWARE_CLASSES=settings.MIDDLEWARE_CLASSES + [
+        'tos.middleware.UserAgreementMiddleware',
+    ]
+)
+class BumpCoverage(TestCase):
```
Comment (nicholasserra) [inline tos/tests/test_middleware.py]: Should we just append these tests to the `TestMiddleware` cases above? Setups seem to be identical?

**Comment by nicholasserra** [comment]
One small nitpick then looks good :D

**Comment by blag** [comment]
Fixed.

**Comment by nicholasserra** [comment]
Thanks for the continued work on this :)

File: tos/tests/test_middleware.py (hunk 1 above)
Comment (blag) [inline tos/tests/test_middleware.py]: I forgot to respond to this, but I split them out because they didn't quite fit into the normal flow of the middleware. I'm happy to create a new PR to merge them together if you like.

Comment (nicholasserra) [inline tos/tests/test_middleware.py]: No worries, they're fine. I may push a more meaningful name for the class to master. :D
